service/controller/resource/promxy/config/config.go

```diff
@@ -0,0 +1,53 @@
+package proxyconfig
```

Comment:
Should that not be package config or promxyconfig?

Reply:
that's how its called upstream, weird but I didn't wanted to fix everything.